src/signals/incident/components/form/MapSelectors/Asset/AssetSelect.test.tsx

```diff
@@ -150,24 +152,34 @@ describe('AssetSelect', () => {
         },
       },
     }
-
-    jest.spyOn(reactRedux, 'useSelector').mockReturnValue(undefined)
   })
 
   afterEach(() => {
     jest.resetAllMocks()
   })
 
   it('should render the Intro', () => {
-    render(withAppContext(<AssetSelect {...props} />))
+    render(
+      withAppContext(
+        <Provider store={setupStore()}>
+          <AssetSelect {...props} />
+        </Provider>
+      )
+    )
 
     expect(screen.queryByTestId('assetSelectIntro')).toBeInTheDocument()
     expect(screen.queryByTestId('assetSelectSelector')).not.toBeInTheDocument()
     expect(screen.queryByTestId('assetSelectSummary')).not.toBeInTheDocument()
   })
 
   it('should render the Selector', () => {
-    render(withAssetSelectContext(<AssetSelect {...props} />))
+    render(
+      withAppContext(
+        <Provider store={setupStore()}>
+          <AssetSelect {...props} />
+        </Provider>
+      )
+    )
 
     userEvent.click(screen.getByText(/kies locatie/i))
```

Comment:
zouden we hier niet een gevolg van het click event moeten testen?

Reply:
dat gebeurt eronder geloof ik, door te checken dat de map verschijnt